src/api/DataSource/types.ts

```diff
@@ -31,6 +31,8 @@ export interface IndexState {
   indexOptimized: boolean;
 }
 
+export type TemporalType = 'date' | 'datetime' | 'localdatetime';
```

Comment:
Using camel case would seem more natural here:

```typescript
export type TemporalType = 'date' | 'dateTime' | 'localDateTime';
```

But I've opted for lowercase instead, for the sake of consistency with the existing enum `PropertyTypeName`:

https://github.com/Linkurious/linkurious-rest-client/blob/c017a74b2bf1cbe363620b266ea8c8bea0e5967f/src/api/GraphSchema/types.ts#L51-L58